addons/account/models/account_move.py

```diff
@@ -991,29 +998,36 @@ def _recompute_dynamic_lines(self, recompute_all_taxes=False, recompute_tax_base
         '''
         for invoice in self:
             # Dispatch lines and pre-compute some aggregated values like taxes.
+            expected_tax_rep_lines = set()
+            current_tax_rep_lines = set()
+            for line in invoice.line_ids:
+                if line.recompute_tax_line:
+                    recompute_all_taxes = True
+                    line.recompute_tax_line = False
+                if line.tax_repartition_line_id:
+                    current_tax_rep_lines.add(line.tax_repartition_line_id._origin)
+                elif line.tax_ids:
```

Comment:
Can't you have both `tax_repartition_line_id` and `tax_ids`?

Reply:
sure when a tax is affecting the base of the subsequent ones

Reply:
Then `elif` doesn't work

Reply:
@smetl is the `elif` on the line below justified if you can have taxes on a repartition line?

Reply:
it's ok no worry. I don't want to account the tax_ids from tax lines here